CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
 
 * Features:
   * Existing Linux node pools can now be updated to enable or disable Federal Information Process Standard (FIPS). See [documentation](https://learn.microsoft.com/azure/aks/enable-fips-nodes#update-an-existing-node-pool-to-enable-or-disable-fips) for more information.
+  * AKS LTS version 1.30 is now generally available.
```

Comment:
Only thing pls add the doc link - https://learn.microsoft.com/en-us/azure/aks/long-term-support . Otherwise good.